File: i2f-jdk/i2f-swl/src/main/java/i2f/swl/cert/data/SwlCertPair.java
package i2f.swl.cert.data;

import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author Ice2Faith
 * @date 2024/8/6 22:14
 * @desc
 */
@Data
@NoArgsConstructor
public class SwlCertPair {
    protected SwlCert server;
    protected SwlCert client;

    public SwlCertPair(SwlCert server, SwlCert client) {
        this.server = server;
        this.client = client;
    }
}

File: i2f-jdk/i2f-swl/web/i2f-swl/src/i2f-turbo-web/i2f-swl/core/cert/data/SwlCertPair.js
/**
 * @param server {SwlCert}
 * @param client {SwlCert}
 * @return {SwlCertPair}
 * @constructor
 */
function SwlCertPair(server=null,
                     client=null) {
    /**
     * 服务端证书
     * @type {SwlCert}
     */
    this.server = server;
    /**
     * 客户端证书
     * @type {SwlCert}
     */
    this.client = client;
}



export default SwlCertPair
